nano/node/bootstrap/bootstrap.cpp

```diff
@@ -732,139 +743,126 @@ bool nano::bootstrap_attempt::process_block_lazy (std::shared_ptr<nano::block> b
 	// Processing new blocks
 	if (lazy_blocks.find (hash) == lazy_blocks.end ())
 	{
-		// Search block in ledger (old)
+		nano::unchecked_info info (block_a, known_account_a, 0, nano::signature_verification::unknown);
+		node->block_processor.add (info);
+		// Search for new dependencies
+		if (!block_a->source ().is_zero () && !node->ledger.block_exists (block_a->source ()) && block_a->source () != node->network_params.ledger.genesis_account)
+		{
+			lazy_add (block_a->source ());
+		}
+		else if (block_a->type () == nano::block_type::state)
+		{
+			lazy_block_state (block_a);
+		}
+		lazy_blocks.insert (hash);
+		// Adding lazy balances for first processed block in pull
+		if (pull_blocks == 0 && (block_a->type () == nano::block_type::send || block_a->type () == nano::block_type::send))
```

Comment:
block_a->type () == nano::block_type::send is repeated twice. Was `state` meant?

Reply:
Sure